CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## 15.5.2
+
+_released `2024-02-05`_
+
+### cli
+
+- :wrench: Updated the "custom-auth" sample to avoid a warning when running zapier validate ([#724](https://github.com/zapier/zapier-platform/pull/724))
+
+- :bug: Fixed bug in `zapier convert` that crashes due to syntax error in user's code and should not replace `source` in sample ([#730](https://github.com/zapier/zapier-platform/pull/730))
+
+- :wrench: Improved handling of broken symlinks while copying files to temp directory during the build process ([#737](https://github.com/zapier/zapier-platform/pull/737))
+
+- :bug: `zapier build` fails with npm workspaces ([#738](https://github.com/zapier/zapier-platform/pull/738))
+
+- :wrench: Added @clif/core dependency, upgraded @oclif/plugin-help ([#739](https://github.com/zapier/zapier-platform/pull/739))
+
+- :bug: Fixed converting triggers breaking when trigger key starts with a number ([#741](https://github.com/zapier/zapier-platform/pull/741))
+
+- :bug: Added comments and fixed a path issue in utils/build.js ([#742](https://github.com/zapier/zapier-platform/pull/742))
```

Comment:
This one can be merged to "`zapier build` fails with npm workspaces". Adding the PR link should be good:

```
:bug: `zapier build` fails with npm workspaces ([#738](https://github.com/zapier/zapier-platform/pull/738), [#742](https://github.com/zapier/zapier-platform/pull/742))
```